service/hook/default_response_provider.go

```diff
@@ -27,13 +27,19 @@ type SuccessRespModel struct {
 // TransformResponseModel ...
 type TransformResponseModel struct {
 	Errors                  []string                             `json:"errors,omitempty"`
+	SkipResponses           []bitriseapi.SkipAPIResponseModel    `json:"skipped,omitempty"`
 	SuccessTriggerResponses []bitriseapi.TriggerAPIResponseModel `json:"success_responses"`
 	FailedTriggerResponses  []bitriseapi.TriggerAPIResponseModel `json:"failed_responses,omitempty"`
 }
 
 // TransformResponse ...
 func (hp DefaultResponseProvider) TransformResponse(input hookCommon.TransformResponseInputModel) hookCommon.TransformResponseModel {
 	httpStatusCode := 201
+
+	if len(input.SuccessTriggerResponses) == 0 && len(input.SkipResponses) > 0 {
+		httpStatusCode = 200
```

Comment:
This was a bit tricky for me also. I would still consider it to be success when skipped instead of a failure. At least 400 sounds weird for me because it usually means that there were some problems with the request. 

Might be that it should be something else than 200 but didn't really find any better alternative when quickly looking at the available status codes.